meta-oe/recipes-core/sdbus-c++/sdbus-c++-tools-native_0.7.2.bb

```diff
@@ -0,0 +1,15 @@
+SUMMARY = "sdbus-c++ native tools"
+DESCRIPTION = "Native interface code generator for development with sdbus-c++"
+
+LICENSE = "LGPLv2.1"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/git-tools/COPYING;md5=1803fa9c2c3ce8cb06b4861d75310742"
+
+inherit cmake native
+
+DEPENDS += "expat"
+RDEPENDS_${PN} = "expat"
```

Comment:
we only need this rdep it if tools are doing dlopen on libexpat, otherwise, shlib tool will automatically pick it up

Reply:
Fixed.